app/jobs/order_cycle_webhook_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Creates a webhook payload for an order cycle
+class OrderCycleWebhookJob < ApplicationJob
+  def perform(order_cycle_id, event)
+    order_cycle = OrderCycle.find(order_cycle_id)
+    data = order_cycle.slice(:id, :name).merge(
+      {
+        orders_open_at: order_cycle.orders_open_at&.to_s,
+        orders_close_at: order_cycle.orders_close_at&.to_s,
+        enterprise_id: order_cycle.coordinator_id,
+        enterprise_name: order_cycle.coordinator.name,
+      }
```

Comment:
Why this translation? Could you just slice all of them? And why to_s?

Reply:
> why to_s?

I can't remember why, but I was making OrderCycleWebhookJob responsible for formatting the dates. I don't think that's necessary, it can be handled by WebhookDeliveryJob (with default formatting anyway).

> Could you just slice all of them?

* `orders_open/close_at`: yep, now with `to_s` removed
* `coordinator_id`: yep, I think it would be better to use the underlying name `coordinator`.
* `coordinator.name`: As far as i know there's no builtin way to 'deep slice', so I think this translation is still necessary.

Reply:
> `coordinator.name`

Oh, of course. I overlooked that one.